### PR TITLE
Add FAQ section

### PR DESCRIPTION
**Pull request by lkluft**

### CODE REVIEW

File: orcestra_book/faq.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+If you are using Jupyter notebooks/lab to run Python code on your machine, you
+can install the `orcestra_book` environment as an ipython kernel.
+
+This way you can select `orcestra_book` as kernel to run your notebooks.
+
+```bash
+conda activate orcestra_book
+ipython3 -m ipykernel install --user --name orcestra_book
+```
```
Comment (yu71ng) [inline orcestra_book/faq.md:13]:
```suggestion
```bash
conda create -n orchestra_book
conda activate orcestra_book
ipython3 -m ipykernel install --user --name orcestra_book
```
```

**Review by yu71ng — CHANGES_REQUESTED** [review]
What exactly is the FAQ for? For technical questions or general FAQs?

File: orcestra_book/faq.md (hunk 1 above)
Comment (lkluft) [reply orcestra_book/faq.md:13]: The idea is to keep it as minimal as possible, so the example assumes `orcestra_book` to be present

**Comment by lkluft** [comment]
> What exactly is the FAQ for? For technical questions or general FAQs?

I don't mind 🤷‍♂️ I was mainly thinking about technical issues, but if you are asked frequently feel free to add 😉

File: orcestra_book/faq.md (hunk 1 above)
Comment (yu71ng) [reply orcestra_book/faq.md:13]: Then maybe mention to install `orchestra_book` into a new environment? (line 6)

```suggestion
```bash
can install the `orcestra_book` into a new environment as an ipython kernel.
```
```

File: orcestra_book/faq.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# FAQ
```
Comment (yu71ng) [inline orcestra_book/faq.md:1]:
```suggestion
# Technical FAQ
```

**Review by yu71ng — APPROVED** [review]

**Pull request merged**